netket/machine/torch.py

```diff
@@ -17,7 +17,8 @@ def __init__(self, module, hilbert):
         self._module = _torch.jit.load(module) if isinstance(module, str) else module
         self._module.double()
         self._n_par = _get_number_parameters(self._module)
-
+        self._parameters = list(self._module.parameters())
```

Comment:
I think that this is likely the origin of the random segfaults, in the old implementation we never store these, because I think they are a temporary object of the torch module. But maybe I am wrong, @twesterhout should know better

Reply:
No, I can't imagine this line having anything to do with segmentation faults... The reason @nikita-astronaut stores a list of `Tensor`s here is because he needs to differentiate with respect to them later on (`_parameters` doesn't have to be a member, could just as well be a local variable... it's a matter of taste).